x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.test.tsx

```diff
@@ -171,7 +172,47 @@ const state: State = {
 };
 
 const { storage } = createSecuritySolutionStorageMock();
-const store = createStore(state, SUB_PLUGINS_REDUCER, kibanaObservable, storage);
+
+const getStoreWithCustomState = (newState: State = state) => {
+  const store = createStore(newState, SUB_PLUGINS_REDUCER, kibanaObservable, storage);
+  return store;
```

Comment:
```suggestion
  return createStore(newState, SUB_PLUGINS_REDUCER, kibanaObservable, storage);
```

💅  

One less `const store` in this file ;)

Reply:
Done.